book/src/continuous_integration/github_actions.md

```diff
@@ -1,7 +1,7 @@
 # GitHub Actions
 
-On the GitHub hosted runners, Clippy from the latest stable Rust version comes
-pre-installed. So all you have to do is to run `cargo clippy`.
+The latest stable version of Rust has Cippy pre-installed on GitHub's hosted runners.
```

Comment:
It is not the latest stable Rust version, that has Clippy pre-installed, it is the GitHub hosted runners that have Clippy pre-installed together with the latest stable Rust version.

The wording can probably still be improved. I wrote that and English is my second language.